### PR TITLE
Added unit test for Barbican service in Ursula

### PR DESCRIPTION
**Pull request by narengan**
Added the tests which will run after Ursula to check if the
barbican service is installed and works as intended.

### CODE REVIEW

**Comment by nirajdp76** [comment]
we should add couple more test around create, show and delete

**Comment by narengan** [comment]
@nirajdp76  Added more tests

File: playbooks/tests/tasks/barbican.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    shell: . /root/stackrc; openstack endpoint list | grep barbican
+
+  - name: Create a barbican secret
+    shell: >
+      . /root/stackrc;
+      openstack secret store --secret-type passphrase --name 'passphrase' --payload 'Pa$$phrasE' -c "Secret href" -f value
+    register: secret_ref
+
+  - name: List barbican secrets
+    shell: . /root/stackrc; openstack secret list
```
Comment (nirajdp76) [inline playbooks/tests/tasks/barbican.yml]: we should verify list show's secret created above.

Also, show add another test show - for secret created.

**Comment by narengan** [comment]
retest

**Comment by narengan** [comment]
retest

**Pull request merged**